deltachat-ffi/src/lib.rs

```diff
@@ -1640,7 +1640,12 @@ pub unsafe extern "C" fn dc_get_blocked_cnt(context: *mut dc_context_t) -> libc:
     }
     let ctx = &*context;
 
-    block_on(async move { Contact::get_all_blocked(&ctx).await.len() as libc::c_int })
+    block_on(async move {
+        Contact::get_all_blocked(&ctx)
+            .await
+            .unwrap_or_default()
```

Comment:
Maybe add `.log_err(&ctx, "Can't get blocked count")` above this line? Also below, where you added `unwrap_or_default()`

Reply:
yip, makes sense. i'll change that.